src/config.rs

```diff
@@ -212,32 +214,55 @@ mod tests {
     }
 
     #[test]
-    fn test_selfavatar_outside_blobdir() -> failure::Fallible<()> {
```

Comment:
This is a `Result` internally. I would keep `?` in the tests instead of `unwrap`s, see https://doc.rust-lang.org/edition-guide/rust-2018/error-handling-and-panics/question-mark-in-main-and-tests.html

Reply:
k, i can revert this. i was just wondering why this is the _only_ test that is different.

Reply:
I just learned about this feature now, probably it is not well-known.